tests/test_package.py

```diff
@@ -57,6 +57,11 @@ def test_package(
                 ["pip", "install", wheel_path], cwd=temp_path, env=venv
             )
 
+            # Install FastAPI into the virtual environment
+            subprocess.check_call(
+                ["pip", "install", "fastapi[standard]"], cwd=temp_path, env=venv
+            )
+
```

Comment:
Can we add a test to make sure we can correctly install `dbos` (without Otel)? I guess that'll be an e2e test.